types/meteor/react-meteor-data.d.ts

```diff
@@ -11,3 +11,17 @@ export function withTracker<TDataProps, TOwnProps>(
  * Requires react-meteor-data 2.0.0 or later
  */
 export function useTracker<TDataProps>(getMeteorData: () => TDataProps, deps?: React.DependencyList): TDataProps;
+
+/**
+ * Requires react-meteor-data 2.4.0 or later
+ */
+export function useSubscribe(name?: string, ...args: any[]): () => boolean;
+// If the factory is non-nullable, it will always return a list
+export function useFind<T = any>(
```

Comment:
I'm not sure if setting `any` as default is the best approach. I think `useFind<T = unknown>` or even `useFind<T>` are better solutions since they don't default to `any`

Reply:
I'd also vote for `useFind<T>`. It shouldn't silently fall back to `any` nor `unknown`, and it's always possible to add `<any>` if needed.

Reply:
I also like that approach